docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -154,6 +154,32 @@ for instance:
     - if you have extended `CountryGridFactory`, revise your changes because class changed its namespace
     - if you have extended `CountryFormType`, revise your changes – new fields are available
     - if you have extended `CountryController` revise your changes – `new` and `edit` actions were added
+- Microservices has been removed and their funcionality has been moved to framework ((#793)[https://github.com/shopsys/shopsys/pull/793/]):
+    - classes from microservices has been moved to `shopsys/framework` package under `Product\Search` namespace
+    - definitions of indexes has been moved to project folder `src/Shopsys/ShopBundle/Resources/definition`, copy them to your project as well
```

Comment:
could you please provide link directly to the json files?

Reply:
well I do not want to, because that link may get obsolete over time, this relative path might "survive" longer :)